lib/std/target.zig

```diff
@@ -631,6 +631,8 @@ pub const Target = struct {
             renderscript64,
             ve,
 
+            // Non-LLVM targets go here
+
```

Comment:
I'd rather modify the implementation of the compiler than mention LLVM in the comments here. Maybe let's not touch this file until adding the SPU Mk II arch.